app/assets/javascripts/oxalis/view/components/button_component.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import React from "react";
+import _ from "lodash";
+import { Button } from "antd";
+
+type ButtonComponentPropType = {
+  onClick?: Function,
+};
+
+/*
+ * A lightweight wrapper around <Button> to automatically blur the button
+ * after it was clicked.
+ */
+class ButtonComponent extends React.PureComponent {
+  props: ButtonComponentPropType;
+  static defaultProps: ButtonComponentPropType = {
```

Comment:
Why do we need a `defaultProp. You are checking for `if (this.props.onClick) {... ` later anyway.